### ISSUE TITLE
the bots aren't even getting the messages in new server's

### ISSUE THREAD

**Issue by CorpseDev** [revoltchat/revite#609]
<p align="center">
<img src="https://user-images.githubusercontent.com/88591830/167275073-071d0a32-eee4-46be-91b5-c11f5ef75956.png" alt="Logo" width="350" height="200">
<img src="https://user-images.githubusercontent.com/88591830/167275075-52713b26-ed81-4dba-abe4-e2e8718bd3c1.png" alt="Logo" width="350" height="200">

**Comment by DoruDoLasu** [comment]
yeah, made tests with creating a new server and inviting the AutoMod bot, sent a message with /help and it didn't respond.
another try with another new server and Taco bot (-help) also didn't succeed, so I can confirm this

**Comment by CorpseDev** [comment]
![image](https://user-images.githubusercontent.com/88591830/167275947-23bfb68e-9740-493b-ae93-21962d94cdaa.png)

**Comment by xXBuilderBXx** [comment]
Yea looks like this is a serious issue i have done some testing it's not just message events!

If a bot gets added to a new server or gets kicked and readded after the bot is connected to the gateway then it stops sending all events for that server.
![VsDebugConsole_TmlBEuhwkv](https://user-images.githubusercontent.com/17956143/167276214-1d24a08a-5b01-4028-a13f-5b01401b1aec.png)
![Reivolt_G2LjNfTy8C](https://user-images.githubusercontent.com/17956143/167276218-4eb4950f-1bdc-46f3-93b9-309ca822b51b.png)

**Comment by Rexogamer** [comment]
(This should be moved to one of the backend repos.)
